package.json

```diff
@@ -21,82 +21,72 @@
     "modal"
   ],
   "scripts": {
+    "build": "npm run clean && babel src -d lib",
+    "build-examples": "rimraf examples/static && webpack --config webpack/docs.config.js",
     "clean": "rimraf lib",
-    "clean:examples": "rimraf examples/static",
-    "build": "npm run clean && babel src --out-dir lib",
-    "build:examples": "npm run clean:examples && webpack --config webpack/docs.config.js",
-    "examples": "npm run clean:examples && babel-node examples/server.js",
-    "lint": "eslint examples/*.js src test *.js",
+    "examples": "webpack-dev-server --config webpack/examples.config.js --hot --port 3000 --progress",
```

Comment:
bizarrely, doing `--hot` here is different from doing `hot: true` in the config. the webpack 1 docs mentioned this (if you did `--hot --inline`), but the webpack 2 docs AFAICT don't